odoo/addons/base/models/ir_actions.py

```diff
@@ -496,28 +496,30 @@ def _default_value(self):
                                  string='Child Actions', help='Child server actions that will be executed. Note that the last return returned action value will be used as global return value.')
     # Create
     crud_model_id = fields.Many2one(
-        'ir.model', string='Target Model',
+        'ir.model', string='Record to Create',
         compute='_compute_crud_model_id', readonly=False, store=True,
-        help="Model for record creation. Set this field only to specify a different model than the base model.")
+        help="Specify which kind of record should be created. Set this field only to specify a different model than the base model.")
```

Comment:
```suggestion
        help="Specify which kind of record should be created.")
```

`crud_model_id` is required when `state == 'object_create'` and invisible if `state != 'object_create'`

maybe we should rename the field `create_model_id` or smtg similar ?

Reply:
you can if you want :D